src/test/java/wooteco/subway/admin/acceptance/LineStationAcceptanceTest.java

```diff
@@ -25,26 +40,128 @@ public static RequestSpecification given() {
     }
 
     /**
-     *     Given 지하철역이 여러 개 추가되어있다.
-     *     And 지하철 노선이 추가되어있다.
-     *
-     *     When 지하철 노선에 지하철역을 등록하는 요청을 한다.
-     *     Then 지하철역이 노선에 추가 되었다.
-     *
-     *     When 지하철 노선의 지하철역 목록 조회 요청을 한다.
-     *     Then 지하철역 목록을 응답 받는다.
-     *     And 새로 추가한 지하철역을 목록에서 찾는다.
-     *
-     *     When 지하철 노선에 포함된 특정 지하철역을 제외하는 요청을 한다.
-     *     Then 지하철역이 노선에서 제거 되었다.
-     *
-     *     When 지하철 노선의 지하철역 목록 조회 요청을 한다.
-     *     Then 지하철역 목록을 응답 받는다.
-     *     And 제외한 지하철역이 목록에 존재하지 않는다.
+     * Given 지하철역이 여러 개 추가되어있다.
+     * And 지하철 노선이 추가되어있다.
+     * When 지하철 노선에 지하철역을 등록하는 요청을 한다.
+     * Then 지하철역이 노선에 추가 되었다.
+     * <p>
+     * When 지하철 노선의 지하철역 목록 조회 요청을 한다.
+     * Then 지하철역 목록을 응답 받는다.
+     * And 새로 추가한 지하철역을 목록에서 찾는다.
+     * <p>
+     * When 지하철 노선에 포함된 특정 지하철역을 제외하는 요청을 한다.
+     * Then 지하철역이 노선에서 제거 되었다.
+     * <p>
+     * When 지하철 노선의 지하철역 목록 조회 요청을 한다.
+     * Then 지하철역 목록을 응답 받는다.
+     * And 제외한 지하철역이 목록에 존재하지 않는다.
      */
     @DisplayName("지하철 노선에서 지하철역 추가 / 제외")
     @Test
     void manageLineStation() {
+        //given
+        Station station1 = createStation("잠실역");
+        Station station2 = createStation("삼성역");
+        Station station3 = createStation("강변역");
+        Line line = createLine("2호선");
+
+        //when 노선 추가
+        LineStationAddRequest lineStationAddRequest1 = new LineStationAddRequest(null, "잠실역", 10, 10);
```

Comment:
```suggestion
        LineStationAddRequest lineStationAddRequest1 = new LineStationAddRequest(null, station1.getName(), 10, 10);
```

이러한 방식으로 최대한 의미있는 값으로 만들어주는 것이 유지보수 관점에서 좋습니다.